### DISCUSSION TITLE
use of http ports

### DISCUSSION THREAD

**Discussion by SprickW** [jhuckaby/Cronicle#637] [Q&A]
I think I followed the description installing master and worker.

Trying to additionally install some web service on a worker server led to a problem. It seems that a worker can also be addressed on port 80 , at least the cronicle web interface shows up.

What is the best way to install cronicle to allow other http based apps on master and worker servers?

Greeting from Europe - Wolfgang

**Comment by jhuckaby** [comment]
Cronicle only lists on the port designated by the `http_port` property in the `config.json` file.  Set that port to whatever you want.  The default is 3012, which shouldn't clash with any other web servers you have running.

Make sure all your Cronicle servers share the same port, so they can communicate properly with each other.

https://github.com/jhuckaby/Cronicle/blob/master/docs/Configuration.md#web-server-configuration

**Comment by SprickW** [comment]
Quick and perfect answer. tx
